src/App.jsx

```diff
@@ -0,0 +1,26 @@
+import React, { useState } from 'react';
+
+import Page from './Page/Page';
+
+function App() {
+  const [state, setState] = useState({
+    count: 0,
+  });
+
+  const { count } = state;
+
+  function handleClick(value) {
+    setState({
+      count: value,
+    });
+  }
```

Comment:
이것도 괜찮은 방법이네요! 👍🏼 

조금만 개선해보면, 이 경우엔 사용하는 컴포넌트들에서 count 상태를 가지고 있어야하는 불편함이 있는 것 같아요.

```suggestion
  function handleClick(value) {
    setState({
      count: count + value,
    });
  }
```
이렇게 처리해보면 어떤가요? 하위 컴포넌트에 넘기는 매개변수를 하나 줄일 수 있게 될 것 같아요. 어떤가요? 😊

다만 이 경우엔 리셋 컴포넌트에서 사용하기 어려워질 수 있겠네요. 어떻게 해결할 수 있을까요?